sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -90,11 +83,11 @@
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>actualAcceleration</EFDB_Name>
-      <Description>Actual acceleration.</Description>
+      <EFDB_Name>actualTorquePercentage</EFDB_Name>
+      <Description>Actual torque for each actuator, as a percentage.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>deg/s^2</Units>
-      <Count>1</Count>
+      <Units>%</Units>
+      <Count>2</Count>
```

Comment:
I see a bit of a naming issue. This presumably comes from motors. LBT uses rear and front, as you have those two on traditional hydrostatic bearing systems (they pull/push against each other to remove backlash).